packages/contracts/src/RelayHub.sol

```diff
@@ -174,6 +176,9 @@ contract RelayHub is IRelayHub, Ownable {
     override
     returns (bool paymasterAccepted, bytes memory returnValue)
     {
+        // #if ENABLE_CONSOLE_LOG
```

Comment:
better find a way to use a macro rather than "ifdef" wrapping.

Reply:
I still think this ifdef wrapping makes using logging cumbersome and far less intuitive

Reply:
solpp can have "macro expansion", e.g.
$$(consoleLog("hello", myvar))
the point is that it will make our source code unusable without solpp - which is bad.
but the alternative of having 3 lines for each comment is also not that great...

Reply:
but you can do multiline stuff like
```
//#ifdef
uint a = 7;
uint b = contractB.getB();
console.log(a, b)
//#endif
```

If you want you can create a macro that would expand, but I don't think we should ban `ifdef` altogether.
This particular log is more of an example than an actual important log.

Reply:
That's exactly what I don't want us to have: creeping conditional code. 
I wish the compiler could verify that all methods are view/pure, but at the minimum, NOT allowing extra code except actual log line is a good thing

Reply:
Well,  I strongly disagree.